modules/ROOT/pages/access-nosql-databases.adoc

```diff
@@ -74,23 +74,13 @@ public MongoClient createMongo() {
     return new MongoClient(new ServerAddress(), creds, new MongoClientOptions.Builder().build());
 }
 ```
-You need to add the `passwordUtilities-1.0` feature, available as a Maven dependency:
-
-```
-<dependency>
-    <groupId>io.openliberty.features</groupId>
-    <artifactId>passwordUtilities-1.0</artifactId>
-    <version>18.0.0.4</version>
-</dependency>
-```
-Additionally, you need to enable the `passwordUtilities-1.0` feature in the `server.xml` file:
+You need to enable the `passwordUtilities-1.0` feature in the `server.xml` file:
 
 ```
 <feature>passwordUtilities-1.0</feature>
 ```
 
-Further, you can enable SSL.
-The `JSSEHelper` gets the `SSLContext`  to create a new `MongoClient` instance and change the `createMongo` method that looks like the following:
+Further, you can enable SSL to change the `createMongo` method that looks like the following:
```

Comment:
Further, you can enable SSL to change the `createMongo` method that looks like the following:

--->

You can enable TLS for your database connection. Add the `JSSEHelper` class to the `createMongo` method to get the `SSLContext` class, as shown in the following example: